### PR TITLE
cli: fix jstp import statement, fixes #129

### LINKED ISSUE #129: Broken import in `tools/cli.js`
It went unnoticed during the review, but I think I broke it in 087d0f5e06ba9921b9959911450522c7af1ede46. Please self-assign if someone is willing to fix it right now, otherwise I'll do that when I get to my editor :)

### PR DESCRIPTION
**Pull request by lundibundi**

### CODE REVIEW

File: tools/cli.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 // TODO: support jstp://server and jstps://server
 
-const jstp = require('.');
+const jstp = require('../');
```
Comment (aqrln) [inline tools/cli.js]: Can you please remove the trailing slash?

**Comment by aqrln** [comment]
@lundibundi and I don't think it is required to request reviews explicitly for such a trivial change ;)

Basically, you've stated that this PR cannot land until both me and @belochub approve it.

**Comment by lundibundi** [comment]
@aqrln ok, done

**Comment by aqrln** [comment]
Landed in a688994cb2dd83193b397f69294f93633aba39d4, thanks!

**Pull request closed without merging**